multi/stm32l4-multi/stm32l4-multi.c

```diff
@@ -160,6 +158,16 @@ static void handleMsg(msg_t *msg)
 			err = uart_write(imsg->uart_set.uart, msg->i.data, msg->i.size);
 			break;
 
+		case otp_get:
+			err = flash_readOtp(imsg->otp_rw.addr, msg->o.data, msg->o.size);
+			break;
+
+		case otp_set:
+			if (imsg->otp_rw.magic == OTP_WRITE_MAGIC) {
+				err = flash_writeOtp(imsg->otp_rw.addr, msg->i.data, msg->i.size);
+			}
```

Comment:
There is not user params check (i.e. if msg->data != NULL)

Reply:
Done